### ISSUE TITLE
"list index out of range" Error for one layer network

### ISSUE THREAD

**Issue by Zepan** [MarsTechHAN/keras2ncnn#30]
~~~
model = Sequential()
model.add(Conv2D(3, (1,1), padding = 'valid',input_shape = (4, 4, 1), use_bias=False));
~~~

convert this model will trigger the error:
~~~
Start emitting to ncnn files.
Traceback (most recent call last):
  File "/home/bits/anaconda3/envs/py374_tf21/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/bits/anaconda3/envs/py374_tf21/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/bits/anaconda3/envs/py374_tf21/lib/python3.7/site-packages/keras2ncnn/__main__.py", line 12, in <module>
    sys.exit(_main())
  File "/home/bits/anaconda3/envs/py374_tf21/lib/python3.7/site-packages/keras2ncnn/keras2ncnn.py", line 81, in main
    graph_seq = emitter.get_graph_seq()
  File "/home/bits/anaconda3/envs/py374_tf21/lib/python3.7/site-packages/keras2ncnn/ncnn_emitter.py", line 18, in get_graph_seq
    q = [graph_head[0]]
IndexError: list index out of range
~~~

Please fix it xiaoxiao!

**Comment by MarsTechHAN** [comment]
Upload the sample model
[face.h5.zip](https://github.com/MarsTechHAN/keras2ncnn/files/6530543/face.h5.zip)

**Comment by MarsTechHAN** [comment]
Fixed in 87357b0
